guess.js

```diff
@@ -20,6 +22,7 @@ function genrateInput() {
         const tryDiv = document.createElement("div")
         tryDiv.classList.add(`try-${i}`, "try")
         tryDiv.innerHTML = `<span>Try ${i} </span>`
+        // [REVIEW] I don't think we need an extra span here (Y).

```

Comment:
u mean we i dont need creat span or writing span on innerHtml

Reply:
Sorry you're right.
I think it's better to use the 2nd approach below ⬇️.

Reply:
massageArea.innerHTML = `game over the word is <span>${wordGuess}</span>`


here if i creat span this text in innrHtml will update to span Contant

Reply:
You already made it work in the OR example below.
```js
        const tryDiv = document.createElement("div")
        tryDiv.classList.add(`try-${i}`, "try")
        // tryDiv.innerHTML = `<span>Try ${i} </span>`
        // OR
        const span = document.createElement("span")
        span.innerHTML = `Try ${i}`
        tryDiv.append(span)
```

Reply:
Tracked in [this ticket](https://github.com/users/AmrHalim/projects/1/views/1?pane=issue&itemId=57468797).